cards/financial-professional-location/component.js

```diff
@@ -6,7 +6,7 @@ class financial_professional_locationCardComponent extends BaseCard['financial-p
   }
 
   onMount() {
-    {{> static/js/interactive-map/location-card/add-card-click-listener}}
+    {{> static/js/interactive-map/location-card/add-card-listeners}}
```

Comment:
could we rename this slightly? When I hear add-card I think of the add card command.

Reply:
outdated due to the refactor